routers/web/repo/actions/view.go

```diff
@@ -310,6 +310,46 @@ func rerunJob(ctx *context_module.Context, job *actions_model.ActionRunJob) erro
 	return nil
 }
 
+func Logs(ctx *context_module.Context) {
+	runIndex := ctx.ParamsInt64("run")
+	jobIndex := ctx.ParamsInt64("job")
+
+	job, _ := getRunJobs(ctx, runIndex, jobIndex)
+	if ctx.Written() {
+		return
+	}
+	if !job.Status.HasRun() && !job.Status.IsRunning() || job.TaskID == 0 {
```

Comment:
I think
```suggestion
	if job.TaskID == 0 {
```
should be enough.

Reply:
Ok, changed it too